programmers/K번째수.js

```diff
@@ -0,0 +1,13 @@
+function solution(array, commands) {
+  let answer = [];
+  let sliceSortArr = [];
+  for (let i = 0; i < commands.length; i++) {
```

Comment:
```suggestion
  let sliceSortArr = [];
  
  for (let i in commands) {
```
for in 문을 사용해볼까요?

그리고 변수 선언한 후에 for문이나 다른 기능을 하는 코드가 있다면 한 칸 띄워주는게 보기 좋아요